cumulus/variables.tf

```diff
@@ -421,3 +421,15 @@ variable "s3credentials_endpoint" {
   default     = false
   description = "Switch that will enable TEA deployment of the /s3credentials endpoint for s3 direct access."
 }
+
+
+variable "html_template_dir" {
+  type        = string
+  default     = null
```

Comment:
I would hope it would. That is what [TEA](https://github.com/asfadmin/thin-egress-app/blob/0c9a53081aea65f235ebe32b5b5e97ef8b87ee49/terraform/variables.tf#L53) sets the default to.

Reply:
I will verify it SIT where we actually have a CloudFront.

Reply:
@mckadesorensen If you could also add the same CHANGELOG entry to the Unreleased section.  That would be great.

Reply:
@mckadesorensen it would be great if we could merge this in before we merge in pr #168.  Did the test work?

Reply:
@lindsleycj I haven't got a chance to yet. It has been a busy week. If the worry is merge conflicts, I can deal with them. If there is another reason, I can find some time to test.

Reply:
@mckadesorensen No problem, I'll proceed with working with Jonathan